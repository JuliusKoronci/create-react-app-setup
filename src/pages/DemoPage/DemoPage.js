// @flow
import React from 'react';
import Demo from '../../modules/Demo/containers/Demo';

const DemoPage = () => (
	<div>
		<h2>DemoPage</h2>
		<Demo />
	</div>
);

export default DemoPage;

