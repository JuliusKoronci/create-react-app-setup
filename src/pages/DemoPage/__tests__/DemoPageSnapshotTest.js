import React from 'react';
import DemoPage from '../DemoPage';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

const wrapper = shallow(<DemoPage />);
describe('(Component) DemoPage', () => {
	it('Should match snapshot', () => {
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});

