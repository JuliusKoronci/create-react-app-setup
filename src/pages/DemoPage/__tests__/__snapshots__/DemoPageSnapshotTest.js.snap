// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) DemoPage Should match snapshot 1`] = `
<div>
  <h2>
    DemoPage
  </h2>
  <Connect(Demo) />
</div>
`;
